packages/amplify-category-auth/provider-utils/awscloudformation/question-factories/core-questions.js

```diff
@@ -42,6 +42,8 @@ function parseInputs(input, amplify, defaultValuesFilename, stringMapsFilename,
   if (input.type && ['list', 'multiselect'].includes(input.type)) {
     if (context.updatingAuth && input.iterator) {
       question = iteratorQuestion(input, question, context);
+      // if selecting existing value to edit it's not require to validate inputs
+      question.validate = () => true;
```

Comment:
Is this set only on update and not on an add?
We have some validations of the URI's on an add which we don't want to loose. Can you please confirm?

Reply:
Yes if the update auth is present in the context, there is an update going on.
![image](https://user-images.githubusercontent.com/56042290/77956052-1616d280-7286-11ea-8479-b376519f3725.png)

Reply:
Yes, but in update also, customers can modify their URL's - we should be validating those URL's as well.